lib/protoflow-blocks/Cargo.toml

```diff
@@ -17,7 +17,7 @@ publish.workspace = true
 [features]
 default = ["all", "std"]
 all = ["hash", "rand", "serde", "sysml", "tracing"]
-hash = ["dep:blake3"]
+hash = ["dep:blake3", "dep:sha2", "dep:sha1", "dep:md-5"]
```

Comment:
Would be a good idea to have these selectable. For example, if I was to use this, I 100% only care about blake3, not the others. In that case, I only want to expect to use blake3.

Reply:
done

Reply:
FYI @joshuajbouw I did not include other hashing algorithms to the 'all' feature.